modules/managed-cluster-notification-policy.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-notifications.adoc
+// * osd_cluster_admin/osd-cluster-notifications.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="managed-cluster-notification-policy_{context}"]
+= Cluster notification policy
+
+Cluster notifications are designed to keep you informed about the health of your cluster and high impact events that affect it.
+
+Most cluster notifications are generated and sent automatically to ensure that you are immediately informed of problems or important changes to the state of your cluster.
+
+In certain situations, Red Hat Site Reliability Engineering (SRE) creates and sends cluster notifications to provide additional context and guidance for a complex issue.
+
+Cluster notifications are not sent for low-impact events, low-risk security updates, routine operations and maintenance, or minor, transient issues that are quickly resolved by SRE.
+
+Red Hat services automatically send notifications when:
+
+* Remote health monitoring detects an issue in your cluster, for example, when a worker node has low disk space.
```

Comment:
@laubai Hi, I have seen the scenarios that automatic notification sent to customer when in-flight checks failed during the cluster installation. Does this covers in mentioned scenarios? I'm not sure if this will be part of "Remote health monitoring" but adding the thoughts for the confirmation.

Reply:
Good question, will check with SRE.

Reply:
Waiting for some info via Slack on this, so @jmekkatt if you have a suggestion for a more general term than "Remote health monitoring or environment verification checks" to use here, let me know!